Microsoft.DotNet.Interactive.PowerShell/StreamHandler.cs

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+using static Microsoft.DotNet.Interactive.Formatting.PocketViewTags;
+
+namespace Microsoft.DotNet.Interactive.PowerShell
+{
+    using System.Collections.Concurrent;
+    using System.Management.Automation;
+
+    internal class StreamHandler
+    {
+        private static readonly PowerShell _pwsh = PowerShell.Create();
+        private static readonly object _pwshLock = new object();
+        KernelInvocationContext _context;
+        IKernelCommand _command;
+        ConcurrentDictionary<int, DisplayedValue> _progresses;
```

Comment:
Please specify `private` and, where possible, `readonly`.

Reply:
fixed.